docs/docsite/rst/user_guide/windows_winrm.rst

```diff
@@ -426,6 +426,8 @@ work. To troubleshoot Kerberos issues, ensure that:
   an alias is being used. The ``krb5.conf`` file needs to be updated so that
   the fully qualified domain name is used and not an alias.
 
+* If the default kerberos tooling has been replaced or modified (some IdM solutions may do this), it may cause issues when installing or upgrading the ``pykerberos`` python library from ``pip``. To resolve this issue, temporarily install the ``krb5-workstation`` and ``krb5-libs`` packages (for RHEL/Fedora), remove any custom kerberos tooling paths from the PATH environment variable, and retry the installation of ``pykerberos``.
```

Comment:
A few points;

* I feel this is still too RHEL specific, we should be referencing the packages we listed in `Installing the Kerberos Library` rather than repeat it here
* I would also not mention `pykerberos` explicitly but say `the Python Kerberos library` as this can change in the future to something else
* MIT is not the only library we know works, some BSD based hosts like MacOS use Heimdal

Because this is a bit of a minefield I'm wondering whether to just have a blanket statement that says this has been tested on MIT krb5 and Heimdal, other solutions may cause issue with authentication and encryption and should be present when running `pip install pywinrm[kerberos]`. Then again, I'm not the best wordsmith so other's could have better ideas.

Reply:
Good points, let's see if this sounds a little better:

> If the default kerberos tooling has been replaced or modified (some IdM solutions may do this), it may cause issues when installing or upgrading the Python Kerberos library.  As of the time of this writing, this library is called `pykerberos` and is known to work with both MIT and Heimdal Kerberos libraries. To resolve `pykerberos` installation issues, ensure the system dependencies for Kerberos have been met (this will link to the prereqs section of the doc), remove any custom Kerberos tooling paths from the PATH environment variable, and retry the installation of ``pykerberos``. 

I think this addresses all of your points and lets people know what we've "generally" tested with and a general approach for solving the issue.